subcommittees/mentoring/procedure-debriefing-session.md

```diff
@@ -0,0 +1,93 @@
+# Procedure - Debriefing Session
+
+This document intends to describe how debriefing sessions
```

Comment:
Don't know how to comment right under @maneesha 's  comment but I would even extend the invitation/notification to be sent to admin-all And if any admin can make it, that;'s great. I think the debrief sessions may help admins in their daily work.